Xamarin.Forms.Platform.Android/Renderers/ListViewRenderer.cs

```diff
@@ -154,9 +155,9 @@ protected override void OnElementChanged(ElementChangedEventArgs<ListView> e)
 					nativeListView = CreateNativeControl();
 					if (Forms.IsLollipopOrNewer)
 						nativeListView.NestedScrollingEnabled = true;
-					_refresh = new SwipeRefreshLayout(ctx);
+					_refresh = new SwipeRefreshLayoutWithFixedNestedScrolling(ctx);
```

Comment:
Let's to this on a virtual method CreateNativePullToRefresh, allow users to just override setting this to a SwipeRefreshLayout or other custom implementation